models/IdeaUpvote.js

```diff
@@ -0,0 +1,25 @@
+const {Model, DataTypes} = require('sequelize');
+const sequelize = require('../config/connection');
+
+class IdeaUpvote extends Model {}
+
+IdeaUpvote.init(
+  {
+    id: {
```

Comment:
We'll want to remove this `id` field, The relations in the `index.js` will configure the needed foreign key columns for us